flow/surface.h

```diff
@@ -35,22 +35,13 @@ class Surface {
 
   virtual bool IsValid() = 0;
 
-  virtual std::unique_ptr<SurfaceFrame> AcquireFrame(const SkISize& size) = 0;
+  virtual std::unique_ptr<SurfaceFrame> AcquireFrame(int64_t view_id,
+                                                     const SkISize& size) = 0;
 
   virtual SkMatrix GetRootTransformation() const = 0;
 
   virtual GrDirectContext* GetContext() = 0;
 
-  virtual std::unique_ptr<GLContextResult> MakeRenderContextCurrent();
-
-  virtual bool ClearRenderContext();
```

Comment:
Could you explain why you removed the context methods from the surface?

Reply:
Because they don't seem needed :)  I was actually testing whether they were, and after I removed these methods it actually compiles.